lxd/storage_pools_config.go

```diff
@@ -103,6 +103,10 @@ func storagePoolFillDefault(name string, driver string, config map[string]string
 	if driver != "dir" {
 		var size uint64
 		if config["size"] == "" {
+			if driver == "lvm" {
+				return nil
+			}
```

Comment:
No, my worries is mostly about the "return nil". If we now add some other default option that we need to fill for LVM it won't be called because of that return.